.cirrus.yml

```diff
@@ -13,7 +13,7 @@ test_task:
       - TEST_OS: fedora-42
       - TEST_OS: centos-9-stream
 
-  fix_kvm_script: sudo chmod 666 /dev/kvm
+  accel_check_script: sudo chmod 666 /dev/kvm
```

Comment:
Note the "fix_kvm" or "accel_check" isn't magic at all. That's just to label the scripts. Only `_script` is magic. so this is a no-op. The thing is that sudo doesn't work any more, or /dev/kvm has wrong permissions (root:pcp 0600)